### ISSUE TITLE
Provide support for scraping metrics for Azure Firewall

### ISSUE THREAD

**Issue by tomkerkhove** [tomkerkhove/promitor#383]
Provide support for scraping metrics for Azure Firewall

**Comment by romainmeunier** [comment]
Any update regarding this ?

**Comment by tomkerkhove** [comment]
Nope, but happy to review contributions! You can already scrape it with the [generic scraper](https://docs.promitor.io/v2.8/scraping/providers/generic-azure-resource/) though.

**Comment by Ghassen95** [comment]
Any update??

**Comment by tomkerkhove** [comment]
No, are you willing to contribute it?
